src/ontology/edits/oeo-model.omn

```diff
@@ -1054,6 +1054,23 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1151",
         <http://purl.obolibrary.org/obo/IAO_0000136> some OEO_00000199
     
     
+Class: OEO_00010262
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A scenario projection is an intentional process in which output (endogenous) data of interest are quantified for future points in time using one or more model calculations based on a scenario.",
```

Comment:
Just a minor remark: I think we can delete the 's' in 'calculations', so it reads: 

A scenario projection is an intentional process in which output (endogenous) data of interest are quantified for future points in time using one or more model calculation based on a scenario.

Reply:
I changed it.